pkg/kernel/networkservice/inject/client.go

```diff
@@ -36,27 +37,33 @@ func NewClient() networkservice.NetworkServiceClient {
 	return &injectClient{}
 }
 
-func (c *injectClient) Request(ctx context.Context, request *networkservice.NetworkServiceRequest,
-	opts ...grpc.CallOption) (*networkservice.Connection, error) {
-	logger := log.FromContext(ctx).WithField("injectClient", "Request")
+func (c *injectClient) Request(ctx context.Context, request *networkservice.NetworkServiceRequest, opts ...grpc.CallOption) (*networkservice.Connection, error) {
```

Comment:
can you also make this change for inject server ?

Reply:
Inject server first make move and after makes Request, so it doesn't need `postpone`.
Or can you please point what changes do you mean?

Reply:
I was talking about this [one](https://github.com/networkservicemesh/sdk-kernel/blob/main/pkg/kernel/networkservice/inject/server.go#L53-L58), but I guess using closeCtx here for ifMoveBack doesn't have any impact here. please just ignore this comment.

Reply:
Thank you for the comment, you are totally right - we should use `postpone` not only for the `Close` but also for closing own resources.

Reply:
Done.